.ci/watchdog/Watchdog.py

```diff
@@ -124,6 +124,15 @@ def run(self, quiet=False):
         self._update_config()
         self._send_message(quiet=quiet)
 
+    def _get_current_time(self):
+        try:
+            now_time = self._git.get_git_time()
+        except Exception:
```

Comment:
What type of exception are you expecting here? 
Can you narrow it down to `Github.GithubException.GithubException`?